tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -487,21 +591,21 @@ def test_get_metric_extraction_config_multiple_widgets_above_max_limit(
             config = get_metric_extraction_config(default_project)
             assert config
 
-            assert capture_exception.call_count == 1
+            assert capture_exception.call_count == 2
             exception = capture_exception.call_args.args[0]
             assert (
                 exception.args[0]
-                == "Spec version 1: Too many (2) on demand metric widgets for org baz"
+                == "Spec version 2: Too many (2) on demand metric widgets for org baz"
```

Comment:
Okay so both spec version fail here and we're just checking the second one now?

Reply:
Both of them fail. I couldn't make up my mind. Perhaps I should not even take the version into consideration.